src/main/kotlin/io/openfuture/state/service/WalletServiceImpl.kt

```diff
@@ -0,0 +1,34 @@
+package io.openfuture.state.service
+
+import io.openfuture.state.controller.domain.dto.WalletDto
+import io.openfuture.state.controller.domain.request.SaveWalletRequest
+import io.openfuture.state.exception.NotFoundException
+import io.openfuture.state.mapper.WalletMapper
+import io.openfuture.state.model.Wallet
+import io.openfuture.state.repository.WalletRepository
+import kotlinx.coroutines.reactive.awaitFirstOrNull
+import kotlinx.coroutines.reactive.awaitSingle
+import org.springframework.stereotype.Service
+import java.time.LocalDateTime
+
+@Service
+class WalletServiceImpl(private val walletRepository: WalletRepository,
```

Comment:
Never align arguments in such a way (https://yadi.sk/i/7mDfTtR3TXLyuw).
In Kotlin it will be:
```
Service(
  val a: A,
  val b: B
)
```

Reply:
`WalletServiceImpl` -> `DefaultWalletService`